README.yaml

```
@@ -39,6 +39,9 @@ related:
   - name: "terraform-null-label"
     description: "Terraform module designed to generate consistent names and tags for resources. Use terraform-null-label to implement a strict naming convention."
     url: "https://github.com/cloudposse/terraform-null-label"
+  - name: "terraform-aws-ssm-patch-management"
+    description: "This module was heavely inspired by @jparnaudeau module"
```

```suggestion
    description: "TAWS Terraform Module to manage SSM Patch Management Resources"
```

Add a separate section to README with the attribution